### PR TITLE
OHRI-2253 Implement pmtct homepage dashboard workflows

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Summary

- This pr restructures pmtct files, showing the demarcation between files in the homepage and patient chart.

## Screenshots

Screenshot 2024-06-21 at 11 24 43

### CODE REVIEW

File: packages/esm-ohri-pmtct-app/src/pmtct/home-dashboard/mother-child-summary-tabs.component.tsx [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import { Tabs, Tab, TabPanels, TabPanel, TabList } from '@carbon/react';
+import styles from './ohri-patient-tabs.scss';
```
Comment (hadijahkyampeire) [inline packages/esm-ohri-pmtct-app/src/pmtct/home-dashboard/mother-child-summary-tabs.component.tsx:3]: Let's move this style import at the bottom of the imports

**Review by hadijahkyampeire — COMMENTED** [review]
I think I missed this but the api.resource file should be taken out of the api folder and put in the src directory just as a file. Otherwise it looks good.

File: packages/esm-ohri-pmtct-app/src/pmtct/home-dashboard/mother-child-summary-tabs.component.tsx (hunk 1 above)
Comment (CynthiaKamau) [reply packages/esm-ohri-pmtct-app/src/pmtct/home-dashboard/mother-child-summary-tabs.component.tsx]: Done

**Review by ODORA0 — APPROVED** [review]

**Review by hadijahkyampeire — APPROVED** [review]
LGTM, thanks @CynthiaKamau

**Pull request merged**